CHANGES.txt

```diff
@@ -31,6 +31,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       we need to do this for internal reasons, those are skipped in that case.
       Bad association could mean some other tool took it over (Visual
       Studio Code is known to do this), or no association at all.
+    - logging code ("debug" call) in Tool/MSCommon now conforms to the
```

Comment:
"Updated debug code is MSVC and MSVS tools to conform to..." ?

Reply:
okay, I changed this to roughly correspond to the suggestion.